src/SiteInformation/FarmDates/EditDatesModal.js

```diff
@@ -0,0 +1,165 @@
+import {useSelector, useDispatch} from 'react-redux';
```

Comment:
do you have auto format on save enabled? there should be a space between `{` and `useSelector`